packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts

```diff
@@ -858,14 +915,21 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
           undefined
         );
       },
+      shouldSkipStartScreen: (context) => {
+        return !!context.componentProps?.disableStartScreen;
+      },
     },
     services: {
       async checkVirtualCameraAndGetStream(context) {
         const { videoConstraints } = context.videoAssociatedParams!;
 
         // Get initial stream to enumerate devices with non-empty labels
+        const existingDeviceId = localStorage.getItem(CAMERA_ID_KEY);
```

Comment:
Let's check with the team on the naming convention of this local-storage key. We would also need to check with appsec maybe?